### ISSUE TITLE
The code is incomplete

### ISSUE THREAD

**Issue by LongLong-Jing** [xanderchf/MonoDepth-FPN-PyTorch#2]
Hi @xanderchf,

Thanks for your great work. It would be a great help if you can you release the full code.

**Comment by LongLong-Jing** [comment]
@xanderchf  Thanks for your replay. I already fixed it.

**Comment by ksnzh** [comment]
The same problem.
Where is the constants and utils file?
@LongLong-Jing How did you fix it.

**Comment by LongLong-Jing** [comment]
@ksnzh 

The function is only for adjusting the learning rate. I will push my code into the projects.

**Comment by 3togo** [comment]
Where is the constants and utils file?
